packages/client/src/client/MultiplexedClient.ts

```diff
@@ -0,0 +1,170 @@
+import { GetProvider } from "@unstoppablejs/provider"
+import type { Client, InteropObservable } from "../types"
+import { createRawClient } from "./RawClient"
+import { ErrorRpc } from "./ErrorRpc"
+import { getInteropObservable } from "../utils/InteropObservable"
+
+type OnData<T = any> = (data: T) => void
+
+interface CacheItem {
+  listerners: Set<OnData>
+  close: () => void
+}
+
+// Do not make it public: https://github.com/ReactiveX/rxjs/pull/6675#pullrequestreview-816542124
+class AbortError extends Error {
```

Comment:
TBH I don't know yet... It's possible that in the future I will add that kind of util under `@unstoppablejs/utils`. However, I _think_ that doing that now it would be a bit of an overkill. Once I notice that I start creating many different custom errors, then yes, I think that I will do something like that, but I rather wait a bit more... I try to avoid [creating abstractions just for the sake of avoiding repetition](https://sandimetz.com/blog/2016/1/20/the-wrong-abstraction), although I do admit that this one really feels like a legit abstraction... So, yeah, probably coming soon :slightly_smiling_face: